docs/api/portable-stories-jest.md

```diff
@@ -245,12 +245,16 @@ The story is prepared by running [`composeStories`](#composestories) or [`compos
 
 ### 3. Load
 
-**(optional)**
-
-Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md). In portable stories, the loaders are not applied automatically—you have to apply them yourself.
+Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md) or [beforeEach](../writing-stories/mocking-modules.md#setting-up-and-cleaning-up). In portable stories, loaders and beforeEach are not applied automatically — you have to apply them yourself.
 
 👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed story will return a `load` method to be called **before** it is rendered.
 
+<Callout variant="info">
+
+While it's technically optional to run `load` before rendering, it is highly encouraged to always do this, even if the story doesn't have any loaders or beforeEach. If you later add any of these to the story, you don't need to remember to also call `load`. Cleaning up previous stories is also important, and calling `load` ensures that later modifying other stories doesn't affect the current story.
```

Comment:
Another big argument is that loaders might come automatically from an addon, and users might not know that. Therefore if you have an addon's loader that is needed for your story to render properly, and you end up not calling load, then you have a broken story.

Reply:
Went with Kyle's prose suggestion.